mlflow/tracking/fluent.py

```diff
@@ -2046,6 +2046,35 @@ def get_logged_model(model_id: str) -> LoggedModel:
     return MlflowClient().get_logged_model(model_id)
 
 
+@experimental
+def last_logged_model() -> Optional[LoggedModel]:
+    """
+    Fetches the most recent logged model in the active experiment.
+    If no logged models are found, returns None.
+
+    Returns:
+        The logged model.
+
+
+    .. code-block:: python
+        :test:
+        :caption: Example
+
+        import mlflow
+
+        assert mlflow.last_logged_model() is None
+
+        model = mlflow.create_logged_model()
+        last_model = mlflow.last_logged_model()
+        assert last_model.name == model.name
+
+    """
+    models = MlflowClient().search_logged_models(
+        experiment_ids=[_get_experiment_id()], max_results=1
+    )
+    return models[0] if models else None
```

Comment:
@harupy This implementation is different from the implementation of `last_active_run()` because `last_active_run()` only considers runs created by the fluent API in the same session (i.e. Python REPL). In contrast, this implementation might return a logged model created by a different process, user, etc.

Can we adjust this implementation so that it only returns that last logged model created by the fluent API (including autologging) from the same session? E.g. maintain a `_last_logged_model` variable and return its value from this API?

Reply:
makes sense, updated!